src/sentry/api/endpoints/project_users.py

```diff
@@ -42,3 +43,30 @@ def get(self, request, project):
             paginator_cls=DateTimePaginator,
             on_results=lambda x: serialize(x, request.user),
         )
+
+    def delete(self, request, project):
+        """
+        Delete an Event User
+        ````````````````````````````````
+
+        Delete an event's user.
+
+        :pparam string organization_slug: the slug of the organization.
+        :pparam string project_slug: the slug of the project.
+        :qparam string query: Prefix ``id`` should be used to match on.
+                              For example, ``query=id:3``
+        """
+        if is_active_superuser(request):
+            identifier = [v for k, v in request.query_params.items()]
+            pieces = identifier[0].strip().split(":", 1)
+            queryset = EventUser.objects.filter(project_id=project.id, ident=pieces[1])
+
+            queryset[0].delete()
+
+            return self.paginate(
```

Comment:
Its somewhat unusual for a delete endpoint to return with paginated results. You could return an empty 200, the client can reload data using the get endpoint afterwards.